### PR TITLE
Add `enableDebugWireframe` to 3D Tiles sandcastles

### PR DESCRIPTION
**Pull request by j9liu**
`ModelExperimental` requires a user to specify `enableDebugWireframe: true` in the constructor of a tileset, in order for the debug wireframe to work. In preparation for enabling `ModelExperimental` by default, this PR adds that flag to the sandcastles that use the 3D Tiles Inspector.

### CODE REVIEW

**Comment by cesium-concierge** [comment]
Thanks for the pull request @j9liu!
* :heavy_check_mark: Signed CLA found.
* :grey_question: [CHANGES.md](https://github.com/CesiumGS/cesium/blob/enable-debug-wireframe/CHANGES.md) was not updated.
    * If this change updates the public API in any way, please add a bullet point to `CHANGES.md`.
* :grey_question: Unit tests were not updated.
    * Make sure you've [updated tests](https://github.com/CesiumGS/cesium/tree/main/Documentation/Contributors/TestingGuide) to reflect your changes, added tests for any new code, and ran the code coverage tool.

Reviewers, don't forget to make sure that:

- [ ] Cesium Viewer works.
- [ ] Works in 2D/CV.

File: Apps/Sandcastle/gallery/development/3D Tiles Split.html [hunk 1]
```diff
@@ -12,7 +12,7 @@
       name="description"
```
Comment (ptrgags) [inline Apps/Sandcastle/gallery/development/3D Tiles Split.html:12]: Ah it looks like there's one more development Sandcastle that needs updating, see `development/Many Clipping Planes`

**Review by ptrgags — COMMENTED** [review]
@j9liu looks good, though I searched through the code and found there's one other sandcastle that uses the 3D Tiles inspector

**Comment by j9liu** [comment]
@ptrgags updated!

**Comment by ptrgags** [comment]
@j9liu looks good!

While reviewing this I discovered that wireframe mode has an index overflow issue, I opened https://github.com/CesiumGS/cesium/issues/10529 for that

**Comment by j9liu** [comment]
Ah I should have mentioned, I found #10527 while working on this PR, for future reference

**Pull request merged**